src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_IntegrationTests.cs

```diff
@@ -10,6 +10,7 @@
 public partial class ThreadPoolBoundHandleTests
 {
     [Fact]
+    [PlatformSpecific(TestPlatforms.Windows)] // ThreadPoolBoundHandle.BindHandle is not supported on Unix
```

Comment:
Oh ok that good to know, in this case though it looks like multiple files are sharing the same class name and the class has tests for both platforms. I'll just leave it as is for now.